prowspecs/specs.go

```diff
@@ -71,6 +71,26 @@ var knownBranches map[string]BranchSpec = map[string]BranchSpec{
 
 		skipTrivy: false,
 	},
+	"release-1.11": {
+		prowContext: &prowgen.ProwContext{
+			Branch: "release-1.11",
+
+			// Freeze test images used.
+			Image: "eu.gcr.io/jetstack-build-infra-images/bazelbuild:20220830-c65cd19-4.2.1",
+
+			// NB: we don't use a presubmit dashboard outside of "master", currently
+			PresubmitDashboard: false,
+			PeriodicDashboard:  true,
+
+			Org:  "cert-manager",
+			Repo: "cert-manager",
+		},
+
+		primaryKubernetesVersion: "1.25",
+		otherKubernetesVersions:  []string{"1.20", "1.21", "1.22", "1.23", "1.24", "1.26"},
```

Comment:
suggestion: This is from the PR being older, but the primary k8s version for 1.11 is 1.26, and 1.25 needs to be added to `otherKubernetesVersions` instead of 1.26!

Reply:
Done.